java/code/src/com/redhat/rhn/frontend/xmlrpc/errata/ErrataHandler.java

```diff
@@ -1152,7 +1152,7 @@ public Errata create(User loggedInUser, Map<String, Object> errataInfo,
 
         ErrataFactory.save(newErrata);
         List<Channel> vendorChannels = channels.stream().filter(Channel::isVendorChannel).collect(toList());
-        if (!vendorChannels.isEmpty()) {
+        if (!vendorChannels.isEmpty() && log.isWarnEnabled()) {
```

Comment:
same comment as debug check

Reply:
This one is to avoid computing ` vendorChannels.stream().map(Channel::getLabel).collect(Collectors.joining(","))` if the log level is not set to WARN